### DISCUSSION TITLE
remarkable firmware 2.6 in the long run

### DISCUSSION THREAD

**Discussion by stheid** [bordaigorl/rmview#62] [Q&A] [answered]
I have seen issue #57 that explains workarrounds to the problem. My question is more general, as i did not fully grasp the problem so far.

**Will rmview at some time in the future support 2.6 out of the box or will the workarround be also needet in the future?**

I am mainly asking to decide whether to hold back updating to 2.6 until rmview supports it or update it and apply the workarround.

**Comment by bordaigorl** [comment, marked as answer]
The issue concerns the RM2 version of the VNC server.
The current workaround is a good way to get it to work without having to modify and recompile the VNC server.
There are a number of alternatives for fixing this on the server side and the author is currently figuring out which is better.
The goal for rmview is to work out of the box on 2.5 and 2.6 for RM1 and RM2.
There is an untested [branch](https://github.com/bordaigorl/rmview/tree/update2.6) which applies the workaround automatically; if the author of the server is too busy to address this (all of this is voluntary work after all) I'll consider incorporating that into the main branch (the problem there is that I don't own an RM2 so I cannot test it myself).

**Reply by stheid** [reply]
All good! did not want to create any pressure or expectation. Really just wanted to know about the logistics. Thats all clear than. Since i am not in urgent need of the 2.6 features i will wait and otherwise work with the workarround ;)

Also thanks for the fast replay ;)
